### PR TITLE
Sessions API 

### PR DESCRIPTION
**Pull request by joey-griffin-cko**
## Sessions API

Added support for SCA payments using `sessions`.

A payment session:
- Identifies and aggregates payment, device and cardholder data in one place
- Is created from a card, network token or a card token
- Can be used as a payment source
- Represents an action when used with a payment (Authentication)

The Sessions API enables the CKO SDKs (JS/Android/iOS) to understand complex SCA flows and react to the state of a payment by using the `next_action` property and hypermedia links.

For backwards compatibility and also for merchants not using our payment interfaces, sessions can be `hosted` which allows the merchant to simply forward the cardholder to a `redirect_url`.

The `auto_complete` property has been added to the `3ds` object to allow the SDKs to complete the payment automatically.

### CODE REVIEW

File: spec/components/schemas/Address.yaml [hunk 1]
```diff
@@ -7,7 +7,11 @@ properties:
   address_line2:
     type: string
     description: The second line of the address
-    example: 90 Tottenham Court Road
+    example: ABC building
+  address_line3:
```
Comment (helder-goncalves-cko) [inline spec/components/schemas/Address.yaml:11]: Is the entire address object used in Sessions? I believe we're using `SessionAddress`

File: spec/paths/sessions@{id}.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+get:
+  tags:
+    - Sessions
+  security:
+    - OAuth:
+      - sessions:app
+      - sessions:browser
+    - ApiKey: []
```
Comment (helder-goncalves-cko) [inline spec/paths/sessions@{id}.yaml:8]: Here we should also be able to use `SessionSecret` correct?

**Comment by joey-griffin-cko** [comment]
Closing this PR, as I rebased the feature branch and it's pulled in a lot of non-session changes.

File: spec/paths/sessions@{id}.yaml (hunk 2 above)
Comment (joey-griffin-cko) [reply spec/paths/sessions@{id}.yaml:8]: Have made the change in this PR https://github.com/checkout/checkout-api-reference/pull/141

**Pull request closed without merging**